src/Adobe.ACPCore.iOS/Adobe.ACPCore.iOS.csproj

```diff
@@ -12,7 +12,7 @@
     <AssemblyName>Adobe.ACPCore.iOS</AssemblyName>
     <RestoreProjectStyle>PackageReference</RestoreProjectStyle>
     <PackageId>Adobe.ACPCore.iOS</PackageId>
-    <PackageVersion>1.0.0</PackageVersion>
+    <PackageVersion>1.0.1</PackageVersion>
```

Comment:
Why is the package version different for iOS and Android Projects? Can we change the android one to 1.0.1?

Reply:
I had a chat with @sbenedicadb and @rymorale about this. Since we upgraded the minor version of the wrapped Android SDK to 1.10, I thought it might be appropriate to release the Android package as v1.1.0, while only updating the patch version for iOS. Versioning across different wrappers is pretty inconsistent, would be good if we can decide on a convention to follow.

Reply:
did we come to a consensus on the versioning?

Reply:
@praveek are we good to release Android as 1.1.0?